frontend/src/utils/data.ts

```diff
@@ -64,6 +64,7 @@ export function summaryStatisticsByServiceAreaAndCensus(
   censusCategoryGroups.forEach(censusGroup => {
     statisticsByGroup[censusGroup] = summaryStatistics(rps, adequacies, { censusCategory, censusGroup })
   })
+  statisticsByGroup[' Total'] = summaryStatistics(rps, adequacies)
```

Comment:
I kinda liked this before more. not sure how "total" looks when it's in with race, sex, etc